packages/labs/virtualizer/src/Virtualizer.ts

```diff
@@ -34,7 +34,8 @@ import {ScrollerController} from './ScrollerController.js';
 // polyfill in the package; this bit of module state facilitates
 // a simple mechanism (see ./polyfillLoaders/ResizeObserver.js.)
 // for loading the polyfill.
-let _ResizeObserver: typeof ResizeObserver | undefined = window?.ResizeObserver;
+let _ResizeObserver: typeof ResizeObserver | undefined =
+  globalThis?.ResizeObserver;
```

Comment:
I don't think the `?` really does anything here just like it doesn't actually do anything for `window?.ResizeObserver` which would break if `window` is not defined in global. Either it's available in global or it's not.

```suggestion
  globalThis.ResizeObserver;
```